hedera-node/src/main/java/com/hedera/services/txns/token/TokenBurnTransitionLogic.java

```diff
@@ -66,13 +69,19 @@ public void doStateTransition() {
 		/* --- Load the model objects --- */
 		final var token = store.loadToken(targetId);
 		final var treasuryRel = store.loadTokenRelationship(token, token.getTreasury());
+		final var ownershipTracker = new OwnershipTracker();
```

Comment:
nit: `ownershipTracker` is not a model object (maybe?). I've added a separate comment in mint for `load trackers` or smth like that.